src/mono/mono/utils/memfuncs.c

```diff
@@ -357,7 +357,9 @@ mono_determine_physical_ram_available_size (void)
 #endif
 
 	if (page_size == -1 || num_pages == -1) {
+#ifndef HOST_WASM
```

Comment:
Looks like unrelated change

Reply:
Yeah this keeps creeping in with local commits because it's broken on main, I'll take it back out